content/en/how-to-guides/configuration-management/version_control.adoc

```diff
@@ -0,0 +1,28 @@
+
+---
+title: "Version Control"
+linkTitle: "Version Control"
+weight: 30
+description: >
+  Version control guidelines
+---
+
+== How to document a part number release ==
+
+Part numbers for CAD, STL, & drawings should adopt the following naming convention: 
+
+  A-TET-ZZZZZZ-R = Assembly 
+      ZZZZZZ-R is Assembly Number & Revision
+  P-TET-ZZZZZZ-R = Part
```

Comment:
According to Patrick's comment [here](https://helpfulengineering.slack.com/archives/G012Q55KWN7/p1595616540008000?thread_ts=1595612022.005500&cid=G012Q55KWN7), the part (and assembly) number is the full X-TET-ZZZZZZ-R. Might be good to check the wording here.

Reply:
Can't seem to access the link, but would it help if there was more verbiage on what a part number is?

Reply:
For now, I think it's okay to leave as it. We're in a churn